src/components/thumbnail/tab-panel-documents-section.tsx

```diff
@@ -105,6 +105,7 @@ export const TabPanelDocumentsSection = observer(({ tab, section, stores, scale,
 
             function handleDocumentClick() {
               onSelectDocument?.(document);
+              section.type === "teacher-supports" && user.setLastSupportViewTimestamp(Date.now());
```

Comment:
```suggestion
              (section.type === "teacher-supports") && user.setLastSupportViewTimestamp(Date.now());
```
Purely a matter of taste, but I'd add the parentheses to help the reader parse the statement correctly.